lib/ansible/modules/crypto/openssl_privatekey.py

```diff
@@ -62,14 +62,30 @@
         required: true
         description:
             - Name of the file in which the generated TLS/SSL private key will be written. It will have 0600 mode.
+    passphrase:
+        required: false
+        description:
+            - The passphrase for the private key.
+        version_added: "2.4"
+    cipher:
+        required: false
+        description:
+            - The cipher to encrypt the private key.
```

Comment:
I think it would be nice to have a idea of the potential value and format, and the default value.